components/home_page/DevfileGrid.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Devfile } from 'customTypes'
+
+import DevfileTile from '@components/home_page/DevfileTile'
+
+import Link from 'next/link'
+import { Gallery } from '@patternfly/react-core'
+
+interface Props {
+  searchDevfiles: Devfile[]
+}
+
+const DevfileGrid = ({ searchDevfiles }: Props) => {
```

Comment:
would add a brief commet for what's supposed to be passed in because name isn't clear

Reply:
Fixed, I renamed to devfile because it just takes the devfiles array